lib/rubycritic/configuration.rb

```diff
@@ -29,7 +29,15 @@ def self.set(options = {})
     end
 
     def self.method_missing(method, *args, &block)
-      configuration.public_send(method, *args, &block)
+      if configuration
```

Comment:
Yeah, lets rather disable the cop.

Then pls squash your PR to one commit afterwards so we cn merge this bad boy ;)